__tests__/compile.test.ts

```diff
@@ -48,3 +53,51 @@ CREATE EXTENSION IF NOT EXISTS pgcrypto WITH SCHEMA public;
 COMMIT;
 `);
 });
+
+it("compiles an included file", async () => {
+  mockFs({
+    "foo.sql": "select * from foo;",
+  });
+  expect(
+    await compile(
+      {
+        connectionString: "postgres://dbowner:dbpassword@dbhost:1221/dbname",
+        placeholders: {
+          ":DATABASE_AUTHENTICATOR": "!ENV",
+        },
+      },
+      `\
+--!include foo.sql
+`,
+    ),
+  ).toEqual(`\
+select * from foo;
+`);
+});
+
+it("compiles multiple included files", async () => {
+  mockFs({
+    "foo.sql": "select * from foo;",
+    "bar.sql": "select * from bar;",
+    "baz.sql": "select * from baz;",
```

Comment:
`--!` include statements should also work inside the fixtures themselves; though we should add a stack of "included files" so we can detect infinite loops.

```suggestion
    "migrations/fixtures/dir1/foo.sql": "select * from foo;",
    "migrations/fixtures/dir2/bar.sql": "select * from bar;",
    "migrations/fixtures/dir3/baz.sql": "--! include dir4/qux.sql",
    "migrations/fixtures/dir4/qux.sql": "select * from qux;",
```

Reply:
haha wow. I had thought of maybe doing this, but figured you wouldn't want to support that level of nesting. Let me see what I can do.